modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from '@emotion/styled-base';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas, {borderRadius} from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React from 'react';
+
+const ActivePage = styled('button')({
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  fontSize: 13,
+  fontWeight: 700,
+  height: 32,
+  width: 32,
+  backgroundColor: canvas.colors.blueberry400,
+  color: canvas.colors.frenchVanilla100,
+  borderRadius: borderRadius.m,
+  ':focus': {outline: 'none'},
```

Comment:
Why is focus being removed from the active page?

Reply:
there was no focus state in the figma, I went ahead and added it back for accessibility purposes.

Reply:
We're having talks with accessibility, design and UX about how and what interactive states should be in designs. Focus rings should never be removed though.